src/jolt/subtable/lower9.rs

```diff
@@ -0,0 +1,59 @@
+use ark_ff::PrimeField;
+use ark_std::log2;
+use std::marker::PhantomData;
+
+use crate::utils::split_bits;
+use std::cmp;
+
+use super::LassoSubtable;
+
+#[derive(Default)]
+pub struct LOWER9Subtable<F: PrimeField> {
+  _field: PhantomData<F>,
+}
+
+impl<F: PrimeField> LOWER9Subtable<F> {
+  pub fn new() -> Self {
```

Comment:
I'd prefer this to be a generic "lower N bits" subtable, so it can be agnostic to `C` and `M`
If we add `C` and `M` as parameters to `JoltInstruction.subtables`, we can compute the number of lower bits we need and pass that in to the constructor here